dropwizard-app/src/test/java/org/coner/util/ApiEntityUtils.java

```diff
@@ -0,0 +1,53 @@
+package org.coner.util;
+
+import org.coner.api.entity.Event;
+import org.coner.api.entity.Registration;
+
+import java.util.Date;
+
+/**
+ *
+ */
+public class ApiEntityUtils {
+
+    protected ApiEntityUtils() {
+        // prevents calls from subclass
+        throw new UnsupportedOperationException();
+    }
```

Comment:
Sorry, this was in response to a checkstyle recommendation that utility classes (ie classes with only static method) should not have a default constructor.  I know ways of preventing this in ruby, but I took the default suggestion of checkstyle to make the constructor protected and to immediately throw an UnsupportedOperationException.  I'm not married this option so I could remove the check all together and removing these constructors.  Thoughts?